cartographer/sensor/map_by_time.h

```diff
@@ -34,6 +34,9 @@ namespace sensor {
 // 'DataType' must contain a 'time' member of type common::Time.
 template <typename DataType>
 class MapByTime {
+ private:
+  using TimeIndexedMap = std::map<common::Time, DataType>;
```

Comment:
No need for this and no better readability? Remove?

Reply:
Okay. I thought that this might make sense since `std::map<common::Time, DataType>` is used 5 times.